simulator/src/main/java/com/github/benmanes/caffeine/cache/simulator/policy/irr/ClockProPlusPolicy.java

```diff
@@ -0,0 +1,663 @@
+/*
+ * Copyright 2015 Ben Manes. All Rights Reserved.
```

Comment:
please update year. You can self-assign if you'd like

Reply:
Done.